docs/framework-app_dynamics_agent.md

```diff
@@ -17,14 +19,14 @@ When binding AppDynamics using a user-provided service, it must have name or tag
 
 | Name | Description
 | ---- | -----------
```

Comment:
@nebhale You're right - prior to AppDynamics Version 4.1 in the minimalist case of a single-tenant on-premise controller, only the controller-host was required and all other parameters were optional or could be derived. However, once Version 4.1 was released, account-access-key has become a mandatory parameter even for single-tenant on-premise controllers.

Spoken to @nkatre about this and he will revert to making all other parameters optional besides controller-host and account-access-key.